webfront/views/entry.py

```diff
@@ -145,6 +143,9 @@ def get(
         *args,
         **kwargs
     ):
+        if endpoint_levels[level - 1].lower() == "tigrfams":
+            full_path = request.get_full_path().lower()
+            return redirect(full_path.replace("tigrfams", "NCBIfam"))
```

Comment:
Keeping URL in lower case for consistency:

```suggestion
            return redirect(full_path.replace("tigrfams", "ncbifam"))
```